.eslintrc.js

```diff
@@ -19,133 +6,20 @@ module.exports = {
-        'src/test/datascience/data-viewing/dataViewerPDependencyService.unit.test.ts',
-        'src/test/datascience/crossProcessLock.unit.test.ts',
-        'src/test/datascience/mockLanguageServerAnalysisOptions.ts',
-        'src/test/datascience/mockLanguageServerProxy.ts',
-        'src/test/datascience/mockPythonSettings.ts',
-        'src/test/datascience/datascienceSurveyBanner.unit.test.ts',
-        'src/test/datascience/color.test.ts',
-        'src/test/datascience/interactive-common/',
-        'src/test/datascience/interactive-common/trustCommandHandler.unit.test.ts',
-        'src/test/datascience/extensionapi/exampleextension/ms-toolsai-test/webpack.config.js',
-        'src/test/datascience/extensionapi/exampleextension/ms-toolsai-test/.eslintrc.js',
-        'src/test/datascience/extensionapi/exampleextension/ms-toolsai-test/src/typings/python.d.ts',
-        'src/test/datascience/extensionapi/exampleextension/ms-toolsai-test/src/serverPicker.ts',
-        'src/test/datascience/mockWorkspaceFolder.ts',
-        'src/test/datascience/kernelFinder.unit.test.ts',
-        'src/test/datascience/mockLanguageClient.ts',
-        'src/test/datascience/notebook/notebookTrust.native.vscode.test.ts',
-        'src/test/datascience/notebook/survey.unit.test.ts',
-        'src/test/datascience/notebook/interrupRestart.native.vscode.test.ts',
-        'src/test/datascience/notebook/contentProvider.native.vscode.test.ts',
-        'src/test/datascience/notebook/edit.native.vscode.test.ts',
-        'src/test/datascience/notebook/saving.native.vscode.test.ts',
-        'src/test/datascience/notebook/notebookEditorProvider.native.vscode.test.ts',
-        'src/test/datascience/notebook/executionService.native.vscode.test.ts',
-        'src/test/datascience/notebook/cellOutput.native.vscode.test.ts',
-        'src/test/datascience/export/exportUtil.test.ts',
-        'src/test/datascience/export/exportToHTML.test.ts',
-        'src/test/datascience/export/exportToPython.test.ts',
-        'src/test/datascience/export/exportManager.test.ts',
-        'src/test/datascience/intellisense.unit.test.ts',
-        'src/test/datascience/interactivePanel.functional.test.tsx',
-        'src/test/datascience/jupyter/serverCache.unit.test.ts',
-        'src/test/refactor/extension.refactor.extract.method.test.ts',
-        'src/test/refactor/extension.refactor.extract.var.test.ts',
-        'src/test/refactor/rename.test.ts',
-        'src/test/workspaceSymbols/provider.unit.test.ts',
-        'src/test/workspaceSymbols/common.ts',
-        'src/test/workspaceSymbols/main.unit.test.ts',
-        'src/test/workspaceSymbols/generator.unit.test.ts',
-        'src/test/datascience/extensionapi/exampleextension/ms-ai-tools-test/src/serverPicker.ts',
-        'vscode.d.ts'
+        '*.js',
+        'vscode.d.ts',
+        '**/*.test.ts',
+        // TODO: Does this file need to be linted?
+        'src/test/pythonFiles/formatting/dummy.ts'
```

Comment:
@rebornix do you know what this file is about? It talks about linting but was previously ignored. I don't know if it's referenced anywhere.

Reply:
I don't know, @DonJayamanne ?